### PR TITLE
Updated 1.3 manifest file for the Alerting and AlertingDashboards plugins

### PR DESCRIPTION
**Pull request by AWSHurneyt**
Signed-off-by: AWSHurneyt <hurneyt@amazon.com>

### Description
Updated 1.3 manifest file for the Alerting and AlertingDashboards plugins
 
### Issues Resolved
https://github.com/opensearch-project/alerting/issues/314
https://github.com/opensearch-project/alerting-dashboards-plugin/issues/179
 
### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: manifests/1.3.0/opensearch-1.3.0.yml [hunk 1]
```diff
@@ -34,7 +34,7 @@ components:
       - gradle:dependencies:opensearch.version: opensearch-ml-plugin
   - name: alerting
     repository: https://github.com/opensearch-project/alerting.git
-    ref: main
+    ref: "1.3"
```
Comment (zelinh) [inline manifests/1.3.0/opensearch-1.3.0.yml:37]: Thanks for the update here! Could you change the double quote here to single quote like `'1.3'`. We used to have `'` so want to be consistent on this. We will take care of the rest for those already merged. Thanks!

**Review by zelinh — APPROVED** [review]
Actually nvm here. I think we should use `""` since it's more widely used in our manifests.

File: manifests/1.3.0/opensearch-1.3.0.yml (hunk 1 above)
Comment (AWSHurneyt) [reply manifests/1.3.0/opensearch-1.3.0.yml:37]: Will do. I'll revise this PR now.

Comment (AWSHurneyt) [reply manifests/1.3.0/opensearch-1.3.0.yml:37]: Never mind, seeing the below comment. Will **not** be revising this PR to use single quotes.

**Comment by AWSHurneyt** [comment]
@zelinh I do not seem to have the necessary permissions to merge this PR. Is that intended, or should I seek the necessary permissions?

**Pull request merged**